### PR TITLE
Added Naruto Website

### LINKED ISSUE #534: Title: Add Naruto Website Using React.
### Is your feature request related to a problem? Please describe.

Website of Naruto made using React and Styled Components.

### Describe the solution you'd like.

A simple website of Naruto. A completely responsive website made using React and styled components that also contain scroll animations implemented using ScrollReveal JavaScript library.

### Describe alternatives you've considered.

For building the user interface, one can also make use of Bootstrap or Material UI instead of Styled components

### Add any other context or screenshots about the feature request here.

_No response_

### PR DESCRIPTION
**Pull request by Bhushan-Thombre**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

YES

### Description

A simple website of Naruto. A completely responsive website made using React and styled components that also contain scroll animations implemented using ScrollReveal JavaScript library.

Fixes #534 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: Web Development/Intermediate/Naruto (React)/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  <img src="Naruto.png" />
+</p>
+
+### Connect With Me
+
+<span>LinkedIn - </span><a href="https://www.linkedin.com/in/bhushan-thombre-209910207/">Click Here</a>
+
+<span>Twitter - </span><a href="https://twitter.com/bhushanat11">Click Here</a>
+
+<span>GitHub - </span><a href="https://github.com/Bhushan-Thombre">Click Here</a>
```
Comment (Kushal997-das) [inline Web Development/Intermediate/Naruto (React)/readme.md:27]: Please reduce the image size .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Bhushan-Thombre write your project name `Naruto` instead of `Naruto-React` Thanks .

**Comment by Bhushan-Thombre** [comment]
I have made the necessary changes.

**Review by Kushal997-das — APPROVED** [review]
@photon149 have look .

**Review by prakhar3062 — APPROVED** [review]

**Pull request merged**